src/openrct2/peep/Peep.cpp

```diff
@@ -4504,80 +4521,69 @@ static void peep_update_ride(rct_peep * peep)
 {
     switch (peep->sub_state)
     {
-    case 0:
-        peep_update_ride_sub_state_0(peep);
+    case PEEP_RIDE_AT_ENTRANCE:
+        peep_update_ride_at_entrance(peep);
         break;
-    case 1:
-        peep_update_ride_sub_state_1(peep);
+    case PEEP_RIDE_IN_ENTRANCE:
+        peep_update_ride_advance_through_entrance(peep);
         break;
-    case 2:
-        peep_update_ride_sub_state_2(peep);
+    case PEEP_RIDE_FREE_VEHICLE_CHECK:
+        peep_update_ride_free_vehicle_check(peep);
         break;
-    case 3:
-        peep_update_ride_sub_state_1(peep);
+    case PEEP_RIDE_LEAVE_ENTRANCE:
+        peep_update_ride_advance_through_entrance(peep);
         break;
-    case 4:
-    {
-        sint16 x, y, xy_distance;
-        if (!peep_update_action(&x, &y, &xy_distance, peep))
-        {
-            peep->sub_state = 5;
-            break;
-        }
-
-        invalidate_sprite_2((rct_sprite *)peep);
-        sprite_move(x, y, peep->z, (rct_sprite *)peep);
-        invalidate_sprite_2((rct_sprite *)peep);
+    case PEEP_RIDE_APPROACH_VEHICLE:
+        peep_update_ride_approach_vehicle(peep);
         break;
-    }
-    case 5:
-        peep_update_ride_sub_state_5(peep);
+    case PEEP_RIDE_ENTER_VEHICLE:
+        peep_update_ride_enter_vehicle(peep);
         break;
-    case 6:
+    case PEEP_RIDE_ON_RIDE:
         // No action, on ride.
         break;
-    case 7:
-        peep_update_ride_sub_state_7(peep);
+    case PEEP_RIDE_LEAVE_VEHICLE:
+        peep_update_ride_leave_vehicle(peep);
         break;
-    case 8:
-        peep_update_ride_sub_state_8(peep);
+    case PEEP_RIDE_APPROACH_EXIT:
+        peep_update_ride_approach_exit(peep);
         break;
-    case 9:
-        peep_update_ride_sub_state_9(peep);
+    case PEEP_RIDE_IN_EXIT:
+        peep_update_ride_in_exit(peep);
         break;
     case 10:
```

Comment:
raw numbers

Reply:
They don't exist. I think I'll just get rid of them all together